public/app/features/canvas/runtime/scene.tsx

```diff
@@ -478,7 +478,7 @@ export class Scene {
         this.selecto.getSelectedTargets()[0].style.cursor = 'grabbing';
       }
 
-      if (isTargetMoveableElement || isTargetAlreadySelected) {
+      if (isTargetMoveableElement || isTargetAlreadySelected || !this.isEditingEnabled) {
```

Comment:
Note: it appears that this blocks the ability to left click select an element (still able to right click / select via the editor menu) - not sure if we care / that it makes sense to select an element on left click anyways when inline editing is disabled (especially if we consider left click actions on canvas elements into the future)